lib/PrintAsObjC/PrintAsObjC.cpp

```diff
@@ -112,6 +112,11 @@ static void writePrologue(raw_ostream &out, ASTContext &ctx) {
          "#else\n"
          "# define SWIFT_NOESCAPE\n"
          "#endif\n"
+         "#if __has_attribute(ns_consumed)\n"
+         "# define SWIFT_RELEASES_ARGUMENT __attribute__((ns_consumed))\n"
+         "#else\n"
+         "# define SWIFT_RELEASES_ARGUMENT\n"
+         "#endif\n"
```

Comment:
I take it it's general policy to use Swift-specific macros here (which we can just add whenever we need them) rather than the NS ones?